### PR TITLE
Feature/prs

### PR DESCRIPTION
**Pull request by buehlere**
Here's a start for #29. 

I've added a slightly modified version of the[ PR template used by nf-core](https://github.com/nf-core/modules/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

I also added a [CODEOWNERS file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) with the [reviewer group](https://github.com/orgs/mskcc-omics-workflows/teams/reviewers/members) as owners of the entire repository. This will add us automatically as reviewers on PRs. Since our repository is configured to require only one approval. It should still hold that only one of us would need to approve.

It's worth noting that nf-core uses [a bash script to auto-generate code-owners](https://github.com/mskcc-omics-workflows/modules/blob/feature/prs/.github/generate_codeowners.sh) for individual modules in the repository. However, they don't have an git-action for it so I'm not exactly sure how / when they run it. I also found it to be a pain to install on git-actions' latest version of ubuntu due to some weird dependencies (`yq` and `fd`). I don't think this is the most necessary feature, but might be nice? You can see my attempt to get this working on this [commit](https://github.com/mskcc-omics-workflows/modules/commit/37930296ad6940102c6199d0df93203a2c8196ea). Let me know your thoughts! 
 
Last thing, I believe the codeowners and the PR template won't take effect until it is merge to the default branch (`main`) so not sure if want to expedite this change to main or not?

### CODE REVIEW

**Comment by anoronh4** [comment]
@buehlere I can see in the "Files Changed" tab that there's issues detected with the CODEOWNERS file. Seems like @mskcc-omics-workflows/reviewers is not valid in the file, even though it definitely seems to be the right handle. I also can't add the team on the reviewer on this PR manually. I think you may have to tweak the settings on the team itself. i don't have permission because i'm not a maintainer for the whole org.

edit: nvm, i was able to go into the settings for this repo and added the team as a "maintainer" -- this role seemed right to me but we can always change it.

**Comment by anoronh4** [comment]
> @buehlere I can see in the "Files Changed" tab that there's issues detected with the CODEOWNERS file. Seems like @mskcc-omics-workflows/reviewers is not valid in the file, even though it definitely seems to be the right handle. I also can't add the team on the reviewer on this PR manually. I think you may have to tweak the settings on the team itself. i don't have permission because i'm not a maintainer for the whole org.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!--
+# mskcc-omics-workflows/modules pull request
+
+Many thanks for contributing to mskcc-omics-workflows/modules!
+
+Please fill in the appropriate checklist below (delete whatever is not relevant).
```
Comment (buehlere) [inline .github/PULL_REQUEST_TEMPLATE.md]: redo folder structure

**Comment by buehlere** [comment]
> @buehlere I can see in the "Files Changed" tab that there's issues detected with the CODEOWNERS file. Seems like @mskcc-omics-workflows/reviewers is not valid in the file, even though it definitely seems to be the right handle. I also can't add the team on the reviewer on this PR manually. I think you may have to tweak the settings on the team itself. i don't have permission because i'm not a maintainer for the whole org.
> 
> edit: nvm, i was able to go into the settings for this repo and added the team as a "maintainer" -- this role seemed right to me but we can always change it.

Thanks for looking into that! The `CODEOWNERS` file now seems valid now. I'm going to promote this from a draft. I think this is at a reasonable state to merge, but happy to make improvements.

**Comment by buehlere** [comment]
> @buehlere I can see in the "Files Changed" tab that there's issues detected with the CODEOWNERS file. Seems like @mskcc-omics-workflows/reviewers is not valid in the file, even though it definitely seems to be the right handle. I also can't add the team on the reviewer on this PR manually. I think you may have to tweak the settings on the team itself. i don't have permission because i'm not a maintainer for the whole org.
> 
> edit: nvm, i was able to go into the settings for this repo and added the team as a "maintainer" -- this role seemed right to me but we can always change it.

Thanks for looking into that! The `CODEOWNERS` file seems valid now. I'm going to promote this from a draft. I think it's at a reasonable state to merge, but also happy to make improvements.

**Review by nikhil — APPROVED** [review]

**Pull request merged**